src/firely-net-sdk.props

```diff
@@ -43,7 +43,7 @@
 		<GenerateDocumentationFile>True</GenerateDocumentationFile>
 		<Configurations>Debug;Release;FullDebug</Configurations>
 		<TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-		<EnablePackageValidation>true</EnablePackageValidation>
+		<EnablePackageValidation>false</EnablePackageValidation> <!-- Disable package validation for now -->
```

Comment:
Let's keep this validation on, so we are stimulated to mention breaking changes in our PRs.